From ee910449bf764d9f582e612c9b8b61b1d18e3a7c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Thu, 9 Apr 2020 14:31:05 +0200
Subject: [PATCH] Adapt to changes in cURL 7.69.0
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

WIN32 macro was removed (1adebe7886ddf20b0733bf9ccbae4ed4866dcfb6) and
then added under a CURL_WIN32 name
(8bd863f97b6c79f561bc063e634cecdf4badf776). This a C preprocessor
macro for driving the C compiler, not a cURL  option. Thus this fix
ignores it.

CURLOPT(na,t,nu) macro was added
(920deff8618a19ae80bd319851722f1b05751f69) as replacement for CINIT()
macro. It's not a cURL option. This fix also ignores it.

CPAN RT#132197

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 Makefile.PL | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.PL b/Makefile.PL
index ad2bd3d..b9e6a46 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -127,7 +127,7 @@ if (!defined($curl_h)) {
     close H;
 
     for my $e (sort @syms) {
-       if($e =~ /(OBSOLETE|^CURL_EXTERN|_LAST\z|_LASTENTRY\z)/) {
+       if($e =~ /(OBSOLETE|^CURL_EXTERN|^CURL_WIN32\z|^CURLOPT\z|_LAST\z|_LASTENTRY\z)/) {
           next;
        }
        my ($group) = $e =~ m/^([^_]+_)/;
-- 
2.21.1

